egs/aishell/s10/chain/train.py

```diff
@@ -29,7 +31,7 @@
 from egs_dataset import get_egs_dataloader
 from model import get_chain_model
 from options import get_args
-
+from device_utils import allocate_gpu_devices
```

Comment:
sort `from` alpahbetically.

In vim, you can

1. visual select a block of lines
2. `:sort`

it will sort automagically for you.